dashboard/app/assets/stylesheets/application.scss

```diff
@@ -74,8 +74,11 @@ body.oceans-blue {
   background-color: $oceans_deep_blue;
 }
 
-body.music-black {
+body.music-black, body.background-dark {
   background-color: $dark_black;
+}
+
+body.background-dark, body.background-light {
   overscroll-behavior: none;
 }
```

Comment:
This ensures that Lab2 levels can all get this fix for an issue in which vertically scrolling on fixed lab content could cause the page header to scroll up and down.  They get this fix once their lesson has dark or light mode selected.  Previously, only music levels had the fix.